common/src/main/scala/com/gu/media/youtube/YouTubeProcessingStatus.scala

```diff
@@ -9,25 +9,26 @@ case class YouTubeProcessingStatus(id: String, status: String, total: Long, proc
                                    timeLeftMs: Long, failure: Option[String])
 
 object YouTubeProcessingStatus {
-  implicit val format: Format[YouTubeProcessingStatus] = Jsonx.formatCaseClass[YouTubeProcessingStatus]
-
-  def apply(video: Video): YouTubeProcessingStatus = {
+  def apply(video: Video): Option[YouTubeProcessingStatus] = {
     val base = YouTubeProcessingStatus(video.getId, status = "", total = 0, processed = 0, timeLeftMs = 0, failure = None)
 
     val summary = video.getStatus
 
     summary.getUploadStatus match {
-      case "uploaded" => parseProcessingStatus(base, video.getProcessingDetails)
-      case "failed" => base.copy(status = "failed", failure = Option(summary.getFailureReason).map(humanizeFailureReason))
+      case "uploaded" =>
```

Comment:
minor: would be good to extract these as case objects, rather than using strings

Reply:
I'd need to have manual JSON Writes then though